components/auth0/connected-accounts.tsx

```diff
@@ -1,19 +1,12 @@
 "use client";
 
-import { useCallback, useEffect, useState } from "react";
 import { useRouter } from "next/navigation";
+import React, { useCallback, useEffect, useState } from "react";
```

Comment:
the only changes on this file are

1. Adding `React` to fix lint error
2. Remove `<Toaster />` component now loaded in higher level at `app/layout.tsx` instead.